python/cudf/cudf/utils/applyutils.py

```diff
@@ -213,11 +216,11 @@ def launch_kernel(self, df, args, chunks, blkct=None, tpb=None):
     def normalize_chunks(self, size, chunks):
         if isinstance(chunks, int):
             # *chunks* is the chunksize
-            return column.arange(0, size, chunks).data_array_view
+            return column.arange(0, size, chunks).data_array_view(mode="write")
         else:
             # *chunks* is an array of chunk leading offset
             chunks = column.as_column(chunks)
-            return chunks.data_array_view
+            return chunks.data_array_view(mode="write")
```

Comment:
Could we avoid going through a `Column` and just directly make cupy arrays in these two cases?

Reply:
Yup, made the change to use cupy arrays directly.